config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  root 'application#hello'
+  root 'login#login'
+  get '/home', to: 'home#home'
+  get '/register', to: 'users#new'
+  post '/register',  to: 'users#create'
+  get '/login', to: 'login#login'
```

Comment:
Can clean this up using `resources :login, only: [:index, :new, :create, :destroy]`